tests/pytorch/test_fused_attn.py

```diff
@@ -666,10 +666,10 @@ def test_transformer_layer(dtype, model_configs, model, ckpt_attn, qkv_format, f
 @pytest.mark.parametrize("dtype", param_types_lean)
 @pytest.mark.parametrize("model_configs", [model_configs_te_layer])
 @pytest.mark.parametrize("model", ["te_1_2", "te_2_0"])
-def test_te_layer_misc(dtype, model_configs, model):
+@pytest.mark.parametrize("qkv_format", ["bshd", "sbhd"])
+def test_te_layer_misc(dtype, model_configs, model, qkv_format):
     """Test TransformerLayer module with miscellanous settings"""
     ckpt_attn = True
-    qkv_format = "bshd"
     fused_qkv_params = True
     RoPE = True
     test_transformer_layer(dtype, model_configs, model,
```

Comment:
I think we need to remove the pytest parameter `qkv_format` on L592 because it's still running `test_transformer_layer` with `sbhd` instead of what's passed in? Might be worth double checking so we're actually testing both scenarios. Thanks!